tests/ValueParsers/QuantityParserTest.php

```diff
@@ -46,9 +46,9 @@ public function validInputProvider() {
 			'2.1250' => QuantityValue::newFromNumber( '+2.1250', '1', '+2.1251', '+2.1249' ),
 
 			'1.4e-2' => QuantityValue::newFromNumber( '+0.014', '1', '+0.015', '+0.013' ),
-			'1.4e3' => QuantityValue::newFromNumber( '+1400', '1', '+1401', '+1399' ),
+			'1.4e3' => QuantityValue::newFromNumber( '+1400', '1', '+1500', '+1300' ),
```

Comment:
I disagree. Numbers like 1400 should have an auto-detected precision of 1, not 100.